src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -153,6 +159,32 @@ typedef struct
     const char* NomDeLaContrainteCouplante;
 } CONTRAINTES_COUPLANTES;
 
+namespace ShortTermStorage
+{
+struct PROPERTIES
+{
+    double capacity;
+    double injectionCapacity;
+    double withdrawalCapacity;
+    double efficiency;
+    std::optional<double> initialLevel;
+
+    std::shared_ptr<Antares::Data::ShortTermStorage::Series> series;
+
+    int globalIndex;
```

Comment:
`int globalIndex` :
I wondered for a short moment what exactly this index stood for.
If it was called **clusterGlobalIndex** (for instance), I would not have.